<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Humble hider</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="path_display">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>No files selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="select_file_button">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Filesystem</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeView" name="fs_tree">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="hide_button">
            <property name="text">
             <string>&amp;Hide</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="recursive_checkbox">
            <property name="text">
             <string>&amp;Recursive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="symlink_checkbox">
            <property name="text">
             <string>Hide &amp;symlink target</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Hidden contents</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeView" name="hidden_view">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="unhide_all_button">
            <property name="text">
             <string>Unhide &amp;all</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="recursive_unhide_checkbox">
            <property name="text">
             <string>&amp;Recursive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="unhide_button">
            <property name="text">
             <string>&amp;Unhide</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>fs_tree</tabstop>
  <tabstop>recursive_checkbox</tabstop>
  <tabstop>hide_button</tabstop>
  <tabstop>path_display</tabstop>
  <tabstop>select_file_button</tabstop>
  <tabstop>hidden_view</tabstop>
  <tabstop>recursive_unhide_checkbox</tabstop>
  <tabstop>unhide_button</tabstop>
  <tabstop>unhide_all_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>path_display</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>manualPathEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_file_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectVictimFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hide_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>hideVictimFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unhide_all_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>unhideAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unhide_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>unhideSolacedFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectVictimFile()</slot>
  <slot>hideVictimFile()</slot>
  <slot>unhideSolacedFile()</slot>
  <slot>scrollFsTreeTo(QString)</slot>
  <slot>unhideAll()</slot>
  <slot>manualPathEntered()</slot>
 </slots>
</ui>
